### ISSUE TITLE
Master doesn't seem compile with Arctic Fox RC 1

### ISSUE THREAD

**Issue by snijsure** [Gurupreet/ComposeCookBook#106]
**Describe the bug**
Master doesn't seem compile with Arctic Fox RC 1

**To Reproduce**

I installed Arctic Fox RC 1 and attempted to compile master  and I see the following error

..../ComposeCookBook/buildSrc/src/main/kotlin/common-compose-module-configs-script-plugin.gradle.kts: (8, 1): Expression 'android' cannot be invoked as a function. The function 'invoke()' is not found

FWIW I am using Android Studio default JDK, anybody else 

<img width="995" alt="Screen Shot 2021-07-24 at 6 25 50 AM" src="https://user-images.githubusercontent.com/1580689/126869837-687470ca-7ce7-4171-863b-710feaa71717.png">

**Comment by Gurupreet** [comment]
@GerardPaligot @hfrsoussama any ideas on this JDK looks correct.

**Comment by Gurupreet** [comment]
@snijsure Anyways currently I am using latest bumblebee canary for latest development.

**Comment by wiryadev** [comment]
I tried it using RC01 with both JDK 11 and 13, compiled perfectly. Maybe try JDK 13?
![image](https://user-images.githubusercontent.com/26060382/126874063-c8c29818-fecd-4664-a010-f746c3c5bf19.png)

**Comment by Gurupreet** [comment]
Thanks @wiryadev :)

**Comment by divyansh-dxn** [comment]
![Screenshot from 2021-07-26 18-01-51](https://user-images.githubusercontent.com/69595691/126989295-584e011a-3bec-43ac-bf04-0ad51c923709.png)
I am unable to run it. It is throwing this error, even I changed JDKs but none of them seems to be working. Any idea??
Also I tried changing compileOptions to VERSION_11.

**Comment by devluckman** [comment]
I tried it using Android Studio Bumblebee 2021.1.1 Canary 7 and I got the same problem, 
I try to downgrade Gradle to version 7.0.0 and kotlin-gradle to version 1.5.10, finally this work for me 

![Screenshot_89](https://user-images.githubusercontent.com/25347788/128985372-6dccd0f8-d489-422a-bf4f-b96ea8de7830.png)

**Comment by zqlq4ever** [comment]
> **Describe the bug**
> Master doesn't seem compile with Arctic Fox RC 1
> 
> **To Reproduce**
> Steps to reproduce the behavior:
> 
> I installed Arctic Fox RC 1 and attempted to compile master and I see the following error
> 
> ..../ComposeCookBook/buildSrc/src/main/kotlin/common-compose-module-configs-script-plugin.gradle.kts: (8, 1): Expression 'android' cannot be invoked as a function. The function 'invoke()' is not found
> 
> FWIW I am using Android Studio default JDK, anybody else
> 
> <img alt="Screen Shot 2021-07-24 at 6 25 50 AM" width="995" src="https://user-images.githubusercontent.com/1580689/126869837-687470ca-7ce7-4171-863b-710feaa71717.png">

me too

**Comment by Gurupreet** [comment]
Hey @zqlq4ever I think I was using latest Bumblebee canary made me update to Gradle. I think as @devluckman suggested please change to "downgrade Gradle to version 7.0.0 and kotlin-gradle to version 1.5.10" should compile in Arctic fox.

Thanks for insights I guess in future I will try to keep it sync with Arctic fox version instead of latest canary 👍🏽

**Comment by GerardPaligot** [comment]
Yes, Arctic Fox need AGP version bigger or equals than 7.0 but slower than 7.1.
And Bumblebee need to use AGP version bigger than 7.1-alpha (current version configured on the project).

@Gurupreet Maybe we need to specify in the README file if this project will be compatible with the Canary or Stable Android Studio version to avoid this kind of issue?

**Comment by alifesoftware** [comment]
@devluckman - Thanks for the solution. 

Would be great if README could be updated with this information.

**Comment by Gurupreet** [comment]
Thanks @GerardPaligot let me update README file for this.

**Comment by Gurupreet** [comment]
Updated please use latest Bumblebee canary 👍🏽
